lib/pbench/server/api/resources/endpoint_configure.py

```diff
@@ -0,0 +1,106 @@
+from flask_restful import Resource, abort
+from flask import jsonify
+
+from pbench.server.api.resources.query_apis import get_index_prefix
+
+
+class EndpointConfig(Resource):
+    """
+    EndpointConfig API resource: this supports dynamic dashboard configuration
+    from the Pbench server rather than constructing a disconnected dashboard
+    config file.
+    """
+
+    def __init__(self, config, logger):
+        """
+        __init__ Construct the API resource
+
+        Args:
+            config (PbenchServerConfig): server config values
+            logger (Logger): message logging
+        """
+        self.logger = logger
+        host = config.get("pbench-server", "proxy_host")
+        self.host = str(host)
```

Comment:
Why do we need to use `str()` here on the value fetched from the configuration file?

Reply:
You've mentioned that before, but I never saw a need. Is the config package really not returning the string?

Reply:
> Why do we need to use `str()` here on the value fetched from the configuration file?

Actually, I'm not sure. That slipped in somehow. I assume the config package returns the string; but I think that in another PR you'd encouraged using `str()` and I got paranoid.